dts/arm/st/u5/stm32u545.dtsi

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 Opito
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/u5/stm32u5.dtsi>
+
+
+/ {
+	soc {
+		/* USB-C PD is not available on this part. */
+		/delete-node/ ucpd@4000dc00;
+
+		compatible = "st,stm32u545", "st,stm32u5", "simple-bus";
+
+		usb: usb@40006000 {
+			compatible = "st,stm32-usb";
+			reg = <0x40006000 0x400>;
+			interrupts = <73 0>;
+			interrupt-names = "usb";
+			num-bidir-endpoints = <8>;
+			ram-size = <1024>;
+			status = "disabled";
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x01000000>,
+				 <&rcc STM32_SRC_HSI48 ICKLK_SEL(0)>;
+			phys = <&usb_fs_phy>;
```

Comment:
If I'm correct, this would be the first non OTG_FS USB IP enabled on U5 and there is none enabled in tree yet, so not tested.
Have you been able to test it on your end ?

Reply:
Yeap! Works great as CDC ACM - though I'll be honest and say I'm not sure how much of the peripheral gets exercised in that configuration.